roles/arteria-sequencing-report-ws/tasks/install.yml

```diff
@@ -26,11 +26,20 @@
     state: present
     extra_args: "-U"
 
+- name: create temp folder
+  tempfile:
+    state: directory
+    suffix: nxf_seqreports
+  register: tempfolder
+
 - name: download seqreports singularity images
   shell: |
     {{ sw_path }}/nextflow/nextflow -log {{ lookup('env', 'SNIC_TMP') }} run {{ arteria_service_src_path }}/seqreports/main.nf -profile test,singularity
   args:
-    chdir: "{{ lookup('env', 'SNIC_TMP') }}"
+    chdir: "{{ tempfolder.path }}"
   environment:
     NXF_SINGULARITY_CACHEDIR: "{{ ngi_containers }}/seqreports"
-    NXF_HOME: "{{ lookup('env', 'SNIC_TMP') }}"
+    NXF_HOME: "{{ nextflow_dest }}/workfiles"
+    NXF_JAVA_HOME: "{{ nextflow_env.NXF_JAVA_HOME }}"
+    NXF_TEMP: "{{ tempfolder.path }}"
```

Comment:
I'm not a 100% sure on these environment variables and the working directory, so if you're wondering "Why did he set it to _that_?", feel free to correct me!

Reply:
In the nextflow role, that's the difference between the `nextflow_local_env` parameters (used when deploying the playbook on miarka3) and the `nextflow_env` parameters (written to config files etc. so they will be in effect when running in the deployed environment on miarka1-2). This looks correct to me though (although I'm often wrong when it comes to these parameters...)